internal/cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -257,9 +256,6 @@ func (c bundleCmd) runMetadata() error {
 	if c.outputDir == "" {
 		c.outputDir = defaultRootDir
 	}
-	if err := os.MkdirAll(c.outputDir, 0755); err != nil {
```

Comment:
Removing this will result in an error being thrown by [`FindBundleMetadata()`](https://github.com/operator-framework/operator-sdk/blob/450a40f070c6735b918d38e2315e68972bcdba57/internal/registry/labels.go#L64), so that case needs to be handled.

Reply:
Have modified to check for metadata only when bundle dir is on disk